webpack/scenes/Subscriptions/components/SubscriptionsTable/components/Table.js

```diff
@@ -34,9 +34,11 @@ const Table = ({
     toggle: ({ rowData }) => toggleSubscriptionGroup(rowData.product_id),
   };
 
+  const collapsibleGroup = Object.values(groupedSubscriptions).some(v => v.subscriptions.length > 1);
```

Comment:
Looks like there's one eslint issue here.  After fixing it you can confirm by running eslint locally from the katello directory like this:
```bash
npx eslint webpack/
```